ghcup-vanilla-0.0.7.yaml

```
@@ -4172,6 +4172,76 @@ ghcupDownloads:
             unknown_versioning:
               dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.1.0/cabal-install-3.10.1.0-aarch64-darwin.tar.xz
               dlHash: fdabdc4dca42688a97f2b837165af42fcfd4c111d42ddb0d4df7bbebd5c8750e
+    3.10.2.0:
+      viPostInstall: "cabal run is currently partially broken on Windows, please see https://github.com/haskell/ghcup-metadata/issues/142."
+      viTags:
+        - Latest
+      viChangeLog: https://github.com/haskell/cabal/blob/master/release-notes/cabal-install-3.10.2.0.md
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31020-64
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-alpine3_12.tar.xz
+              dlHash: b54e1cd235c47c62c03cdb9f6cf90e5fe8ae38c5e5befb9e21c8d1395f4bde05
+          Linux_Alpine:
+            unknown_versioning: *cabal-31020-64
+          Linux_CentOS:
+            unknown_versioning: &cabal-31020-64-centos7
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-centos7.tar.xz
+              dlHash: cfcdab399380dec7fedda55898bff975ac30b5d5d579433cbf8773b17c15f410
+          Linux_Debian:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb10.tar.xz
+              dlHash: bdeb27c008b09c3b86f8a2768018d62a1aee02565304d123fda87ed432549418
+          Linux_Fedora:
+            '( >= 27 && < 33)': *cabal-31020-64-centos7
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-fedora33.tar.xz
+              dlHash: 1e59dc1e1a1b33085a1789b8ddafb55026211454efe0b4e814c956ef86fe3ea5
+          Linux_Ubuntu:
+            '( >= 16 && < 19 )': &cabal-31020-64-ubuntu18
```

On second thought... this probably should be

```suggestion
            '< 19': &cabal-31020-64-ubuntu18
```

There's very little chance that ubuntu older than 18 will work with a ubuntu 20 bindist